src/lib/core/partialcubeop.ts

```diff
@@ -1,45 +1,70 @@
-import { CubeIndex } from "lib/core/cubeindex";
+import { CubeIndex, Face } from "lib/core/cubeindex";
 
 /**
- * Computes the new (row, column) coordinates of the input CubeIndex after a
- * clockwise rotation. Does not account for movements of indices on adjacent faces.
+ * CubeIndex literals will be represented as {FRC} where:
+ * F is the face coordinate
+ * R is the row coordinate
+ * C is the column coordinate
+ */
```

Comment:
Going to move this and other docs about the cube representation to the main README at some point.

Reply:
link out to a task 🙏

Reply:
Done.